README.md

```diff
@@ -132,6 +149,14 @@ To learn more, check out the documentation and/or the source code.
 * [ruleguard package](https://godoc.org/github.com/quasilyte/go-ruleguard/ruleguard) reference
 * Introduction article: [EN](https://quasilyte.dev/blog/post/ruleguard/), [RU](https://habr.com/ru/post/481696/)
 
+## Ready to use rules
+
+There is the list of open-source rules you can use in your project:
+
+* [go-ruleguard/rules.go](https://github.com/quasilyte/go-ruleguard/blob/master/rules.go)
+* [semgrep-go/ruleguard.rules.go](https://github.com/dgryski/semgrep-go/blob/master/ruleguard.rules.go)
+* [go-critic/rules.go](https://github.com/go-critic/go-critic/blob/master/rules.go)
```

Comment:
`gocritic` specific rules, there is no point to share them with others.

Reply:
Fixed :+1: